packages/garbo/src/config.ts

```diff
@@ -82,18 +83,28 @@ function stringToWorkshedItem(s: string): Item | null {
   return validWorksheds[0].item;
 }
 
-const defaultTarget =
-  ChestMimic.have() ||
-  have($item`Clan VIP Lounge key`) ||
-  (CombatLoversLocket.have() &&
-    CombatLoversLocket.unlockedLocketMonsters().includes(
-      $monster`cheerless mime executive`,
-    ))
-    ? $monster`cheerless mime executive`
-    : maxBy(
-        $monsters.all().filter((m) => m.wishable && isFreeAndCopyable(m)),
-        valueDrops,
-      ) || $monster`Witchess Knight`;
+function defaultTarget() {
+  if ($skills`Curse of Weaksauce, Saucegeyser`.every((s) => have(s))) {
+    if (
+      ChestMimic.have() ||
+      have($item`Clan VIP Lounge key`) ||
+      (CombatLoversLocket.have() &&
+        CombatLoversLocket.unlockedLocketMonsters().includes(
+          $monster`cheerless mime executive`,
+        ))
+    ) {
+      return $monster`cheerless mime executive`;
+    } else {
+      return (
+        maxBy(
+          $monsters.all().filter((m) => m.wishable && isFreeAndCopyable(m)),
+          valueDrops,
+        ) || $monster`Witchess Knight`
+      );
+    }
+  }
+  return $monster`Knob Goblin Elite Guard Captain`;
+}
 
 export const targettingMeat = () =>
   $monsters`cheerless mime executive`.includes(globalOptions.target);
```

Comment:
this needs to consider guard capt. maybe just have it check monster base meat is above some threshhold and not free